exercises/protein-translation/tests/proteins.rs

```diff
@@ -0,0 +1,97 @@
+extern crate protein_translation as proteins;
+
+#[test]
+fn test_methionine() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("ATG"), Ok("methionine"));
+}
+
+#[test]
+#[ignore]
+fn test_cysteine_tgt() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TGT"), Ok("cysteine"));
+}
+
+#[test]
+#[ignore]
+fn test_cysteine_tgy() { // "compressed" name for TGT and TGC
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TGY"), Ok("cysteine"));
+}
+
+#[test]
+#[ignore]
+fn test_stop() {
+    let info = proteins::parse(make_pairs());
+    assert_eq!(info.name_for("TAA"), Ok("stop codon"));
```

Comment:
This is probably just me not understanding the example code, but I don't see anything in the example that could return `Ok("stop codon")`